lighthouse-plugin-publisher-ads/package.json

```diff
@@ -30,6 +30,6 @@
     "husky": "^1.3.1",
     "mocha": "^5.1.1",
     "sinon": "^5.0.7",
-    "typescript": "^2.9.1"
+    "typescript": "^3.5.1"
```

Comment:
Did you upgrade this with yarn? I would think the yarn.lock file should have been updated.

Reply:
I did. Perhaps it's not included because it's a dev dependency? I see a version of typescript in yarn.lock, but that appears to be have been brought in from one of our dependencies

Reply:
Turns out I forgot to run yarn in the root folder. Updated both package.json files. Thanks for catching that Jon!